README.md

```diff
@@ -417,6 +417,81 @@ To activate this mode, add in your HTML code the web component with the followin
+  - When all the rules have been respected and the position is available but after an action, a rule is violated. In this case, the payload of the event is undefined
+    - event.detail example: undefined
+
+## Create and Select Mode
+
+To activate this mode, add in your HTML code the web component with the following parameters:
+
+```
+ <openlayers-element  options='{ "enableDraw": false, 
+        "information": { "duration": 5000, "title": "Signaler un banc cassé", "content": "Positionner le centre de la cible à l’emplacement du banc cassé dans l’espace public" },
+        "enableGeolocation": true,
+        "enableCenterButton": true,
+        "enableRotation": true,
+        "mode": {
+            "type": "mix"
+        },
+        "geolocationInformation": {
+            "displayBox": true,
+            "reverseLocation": true,
+            "currentLocation": false
+        },
+        "notifications": [
+        {
+            "type": "warning",
+            "message": "Veuillez zoomer davantage avant de pouvoir sélectionner un emplacement.",
+            "rule": {
+                "type": "ZOOM_CONSTRAINT",
+                "minZoom": 16
+            }
+        },
+        {
+          "type": "warning",
+          "message": "Le maximum de sélection est limité à {x}.",
+          "rule": {
+              "type": "MAX_SELECTION",
+              "maxElement": 3
+          }
+        },
+        {
+            "type": "info",
+            "message": "Déplacez la carte pour que l’endroit désiré soit au centre de la cible",
+                "rule": {
+                "type": "MOVE_TARGET"
+            }
+        }
+        ],
+        "wfs": {
+            "url": "https://mapnv.ch/mapserv_proxy?ogcserver=source+for+image%2Fpng&SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=ELE_tragwerk_gesco"
+        },
+        "wmts": [{
+            "capability": "https://wmts.asit-asso.ch/wmts?&Service=WMTS&Version=1.0.0&Request=GetCapabilities",
+            "layer": "asitvd.fond_cadastral",
+            "projection": "EPSG:2056",
+            "name": "Carte de base",
+            "thumbnail": "http://localhost:5173/base.svg"
+        },
+        {
+            "capability": "https://wmts.geo.admin.ch/EPSG/2056/1.0.0/WMTSCapabilities.xml",
+            "layer": "ch.swisstopo.swissimage",
+            "projection": "EPSG:2056",
+            "name": "Photo aérienne",
+            "thumbnail": "http://localhost:5173/aerial.svg"
+        }],
+        "selectionTargetBoxMessage": "Emplacement du banc"
+    }'
+/>
+```
+
+### Validation events
+
+For this scenario, there is one events to listen:
+
 - `position-selected`: This event is sent in two cases:
   - When all the rules are met and the position is available. The information is stored in an array of object in event.detail and this object contains only the geometry in WTK format.
     - [{ geometry: POINT (2538545.7462747833 1180732.9753953428)) }];
```

Comment:
there is no id in case of selected object?

Reply:
Right, modification done